xesmf/frontend.py

```diff
@@ -28,6 +29,30 @@
     dask_array_type = ()
 
 
+def subset_regridder(
+    ds_out, ds_in, method, in_dims, out_dims, locstream_in, locstream_out, periodic, **kwargs
+):
+    """Compute subset of weights"""
+    kwargs.pop('filename', None)  # Don't save subset of weights
+    kwargs.pop('reuse_weights', None)
+
+    # Renaming dims for the subset regridding
```

Comment:
Could you give more explanations on the need for renaming dimensions.

Reply:
Before map_blocks, dimensions are renamed to explicitly state from what dataset they come from. This prevents the case where the two datasets could have e.g. lon/lat as dimensions which would prevent map_blocks from correctly identifying the blocks to pass to the function. I just did some checks and the renaming happening in subset_regridder does not seem to be necessary, I'll remove it.

Reply:
I mean in the code ! ; )